### PR TITLE
Add uri to user resources for MongoDB Flex and PostgreSQL Flex

### PR DESCRIPTION
**Pull request by GokceGK**

### CODE REVIEW

File: stackit/internal/services/mongodbflex/user/resource.go [hunk 1]
```diff
@@ -179,6 +180,10 @@ func (r *userResource) Schema(_ context.Context, _ resource.SchemaRequest, resp
 			"port": schema.Int64Attribute{
 				Computed: true,
 			},
+			"uri": schema.StringAttribute{
+				Computed:  true,
+				Sensitive: true,
+			},
 		},
```
Comment (vicentepinto98) [inline stackit/internal/services/mongodbflex/user/resource.go:187]: you should also extend the warning on line 337 stating that the imported resource will have an empty uri field

File: stackit/internal/services/postgresflex/user/resource.go [hunk 2]
```diff
@@ -185,6 +186,10 @@ func (r *userResource) Schema(_ context.Context, _ resource.SchemaRequest, resp
 			"port": schema.Int64Attribute{
 				Computed: true,
 			},
+			"uri": schema.StringAttribute{
+				Computed:  true,
+				Sensitive: true,
+			},
 		},
 	}
 }
```
Comment (vicentepinto98) [inline stackit/internal/services/postgresflex/user/resource.go:195]: you should also extend the warning on line 341 stating that the imported resource will have an empty uri field

File: stackit/internal/services/mongodbflex/user/resource.go (hunk 1 above)
Comment (GokceGK) [reply stackit/internal/services/mongodbflex/user/resource.go:187]: thank you :) extended the warning

File: stackit/internal/services/postgresflex/user/resource.go (hunk 2 above)
Comment (GokceGK) [reply stackit/internal/services/postgresflex/user/resource.go:195]: thank you :) extended the warning

**Review by vicentepinto98 — APPROVED** [review]

**Pull request merged**